sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -374,6 +374,15 @@ def _from_batch(cls, batch_data, partition_key=None):
         )
         return batch_data_instance
 
+    def _load_events(self, events):
+        for event_data in events:
+            try:
+                self.add(event_data)
```

Comment:
Should we validate event_data type here, or is EH flexible/do we get a sane error later?  (Thinking in terms of in SB where if someone tries to add a string where it's enumerable but not a valid Message type)

Reply:
The user will finally get an AttributeError because some code will finally call event_data.message.